src/bot.ts

```diff
@@ -141,9 +142,14 @@ export function createBot(config: BotConfig) {
       return path.resolve(config.dataDir, `${content}.mp3`);
     }
     if (config.myInstantsEnabled && content.startsWith("instant")) {
-      const sound = encodeURIComponent(content.substr("instant".length).trim().substr(0, 256))
+      const sound = encodeURIComponent(content.slice("instant".length).trim().slice(0, 256))
       return `https://www.myinstants.com/media/sounds/${sound}.mp3`;
     }
+    if (config.wikiaCDNEnabled && content.startsWith("wikiacdn")) {
```

Comment:
you can just make it wikia. 

For that matter, we should also had this configurable from the start. This PR is fine but if you want you can make a followup that reads command names from `config.json`. Defaults could be "instant", "wikiacdn/wikia" whatev.